dspace-api/src/main/java/org/dspace/discovery/configuration/DiscoveryConfigurationService.java

```diff
@@ -75,6 +76,17 @@ public DiscoveryConfiguration getDiscoveryConfigurationByNameOrDso(final String
         }
     }
 
+    public List<DiscoveryConfiguration> getIndexAlwaysConfigurations() {
```

Comment:
Missing Javadocs.  This is a straightforward method, but we should get in the habit of adding Javadocs everywhere.

Reply:
I've included the Javadocs